### ISSUE TITLE
Several question about wan CausVid and fp8 model

### ISSUE THREAD

**Issue by thbupt** [kijai/ComfyUI-WanVideoWrapper#581]
Great work! I have several questions. 
1. How was the model [Wan21_CausVid_14B_T2V_lora_rank32.safetensors] trained? Was the training conducted with reference to (https://huggingface.co/lightx2v/Wan2.1-T2V-14B-CausVid)？Could you share some training details, such as the size of training data.
2. How is FP8 quantization performed? Are there any reference papers?

**Comment by thbupt** [comment]
> > Great work! I have several questions.
> > 
> > 1. How was the model [Wan21_CausVid_14B_T2V_lora_rank32.safetensors] trained? Was the training conducted with reference to ([https://huggingface.co/lightx2v/Wan2.1-T2V-14B-CausVid)？Could](https://huggingface.co/lightx2v/Wan2.1-T2V-14B-CausVid)%EF%BC%9FCould) you share some training details, such as the size of training data.
> > 2. How is FP8 quantization performed? Are there any reference papers?
> 
> It's simply a LoRA extracted from those weights you linked, as in the difference between original Wan 14B T2V and the CausVid 14B, reasoning for this is to allow only utilizing their distillation without the causal aspects so it can be used with our existing workflows and with stuff like VACE.
> 
> With fp8 I'm not doing anything special, still a lot to learn about doing that properly myself, for now I've just been casting the weights to torch.float8_e4m3fn, which works surprisingly well with Wan but is obviously very loss and suboptimal thing to do.

@kijai Thanks for your reply, I'm still a bit confused about the lora extraction. Is it similar to performing SVD decomposition on the weight difference between the two models for linear layers (maybe directly compute the bias difference)?
